themes/SuiteP/css/editview.scss

```diff
@@ -2631,10 +2631,212 @@ input#btn_ActionLine.button[type="button"] {
   background: none;
 }
 
+#fieldLines {
+  display: table;
+  width: 100%;
+}
+
 #fieldLines_head {
   background: none;
 }
 
+thead#fieldLines_head > tr > td {
+  padding-right: 8px;
+  background-color: $default-tab-bg;
+  padding-top: 8px;
+  padding-bottom: 8px;
+}
+// remove button
+thead#fieldLines_head > tr > td:first-of-type {
+  border-top-left-radius: 4px;
+}
+
+thead#fieldLines_head > tr > td:nth-of-type(1) {
+  width: 40px;
+  height: 32px;
+}
+// module
+thead#fieldLines_head > tr > td:nth-of-type(2) {
+  width: 10%;
+}
+//
+thead#fieldLines_head > tr > td:nth-of-type(3) {
+  width: 10%;
+}
+
+thead#fieldLines_head > tr > td:nth-of-type(5) {
+  width: 5%;
+}
+
+
+thead#fieldLines_head > tr > td:nth-of-type(4) {
+  width: 5%;
+}
+
+thead#fieldLines_head > tr > td:nth-of-type(6) {
+  width: 15%;
+}
+
+thead#fieldLines_head > tr > td:nth-of-type(7) {
+  width: 10%;
+}
+
+thead#fieldLines_head > tr > td:nth-of-type(8) {
+  width: 10%;
+}
+
+thead#fieldLines_head > tr > td:nth-of-type(9) {
+  width: 5%;
+}
+
+thead#fieldLines_head > tr > td:nth-of-type(10) {
+  width: 10%;
+}
+
+thead#fieldLines_head > tr > td:nth-of-type(11) {
+  width: 10%;
+}
+
+[aor-field-line-body]:first-of-type td {
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+  border-top: none;
+  border-bottom: 4px solid $main-alternate-bg;
+}
+
+[aor-field-line-body] td  {
+  background-color: $detail-view-field-bg;
+  border-top: 2px solid $main-alternate-bg;
+  border-bottom: 2px solid $main-alternate-bg;
+  border-top-left-radius: 4px;
```

Comment:
The border-radius is needed to keep to the same aesthetic as SuiteP - which employ rounded corners for more elements. It's a design choice. Plus it helps to visually separate the lines.